python3: can't open file '/g/data/e14/as3189/OFAM/scripts/create_file_wsc.py': [Errno 2] No such file or directory
